### ISSUE TITLE
web/index.html constantly updating

### ISSUE THREAD

**Issue by mrgloom** [junyanz/pytorch-CycleGAN-and-pix2pix#846]
If I open web/index.html via firefox browser it constantly updating and it causes flickering, is there any way to fix it?

**Comment by mrgloom** [comment]
Looks like it can be fixed here https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/18a40e606eb5ef5214db84b1bb24b9f0e3641371/util/html.py#L20

**Comment by junyanz** [comment]
Yes, you can use the `--refresh` flag.

**Comment by mrgloom** [comment]
Setting some large number in `web/index.html` helps:

```
<!DOCTYPE html>
<html>
  <head>
    <title>Experiment name = maps_pix2pix</title>
    <meta content="100000" http-equiv="refresh">
  </head>
```

**Comment by Hunterwolf88** [comment]
Hey guys, I have this problem too...

In the file "pytorch-CycleGAN-and-pix2pix/util/html.py" I have that line, as suggested by @mrgloom  but it looks just like a comment. Where the actual variable is defined?

I also tried editing the "web/index.html" file but I lost the edit everytime the training process updates it.

Where can I use the "--refresh" flag? I tried to add "--refresh 0" in the script that runs my train.py (together with --name, --model, --continue_train, --epoch_count and so on) but it's considered unknown.
